docsrc/imap/download/release-notes/3.13/x/3.13.1.rst

```diff
@@ -0,0 +1,79 @@
+:tocdepth: 3
+
+===========================
+Cyrus IMAP 3.13.1 Tag Notes
+===========================
+
+Unavailable for download as this is a development branch only.
+
+Access is via git.
+
+.. warning::
+
+    This should be considered for
+    **testing purposes** and **bleeding-edge features** only. We will try to
+    tag these snapshots at coherent development points, but there will
+    generally be **large breaking changes** occurring between releases in this
+    series.
+
+.. _relnotes-3.13.1-changes:
+
+Major changes since the 3.12 series
+===================================
+
+* The annotator is now called only on items added to email-type mailboxes.
+* There is now a ``--with-zoneinfo-dir`` option to configure.
+* ``ctl_conversationsdb`` now has ``-U`` for version upgrades.
+* ``lmtpd`` now provides a capability called ``TRACE`` and a new ``TRACE``
+  command for setting an interal trace-id, which will be logged.
+* The ``admins_get_internal_seen`` configuration option has been added.  If
+  set, the admin users will always get and set the same ``\Seen`` flags as the
+  mailbox owner.
+* ``master(8)`` will refuse to start if multiple partitions appear to be
+  identical or overlap.
+* Cyrus syslogs are in the process of being converted to ``logfmt``, a
+  structured logging format.  So far, this affects primarily the login logs.
+* The ``backupcyrusd`` protocol daemon has been added to facilitate rolling
+  backups.
+* The ``expire_by_savedate`` and ``expire_keep_flagged`` configuration options
+  have been added to control the behavior of ``cyr_expire``.
+* Time zone data is no longer included in iTIP messages.
+* The IMAP PREVIEW extension will now only be advertised if
+  the ``jmap_preview_annot`` configuration option is set and begins with
+  ``/shared/``.
+* Support for the IMAP ``UIDBATCHES`` extension has been added.
+* ``cyrus.conf`` can now schedule ``EVENTS`` with cron-like syntax.
+* The new ``sieve_mdn_original_recipient_header`` configuration option allows
+  an install to get the original recipient from a header other than
+  Original-Recipient.
+* The new ``sieve_mdn_private`` configuration option helps prevent leaking
+  usernames.
+* A new database backend called ``twom`` has been added.
+
+
+Updates to default configuration
+================================
+
+* OpenSSL version 3.0 or greater is now required for all builds.
+
+Removed features
+================
+
+* The undocumented ``XSTATS`` metrics system has been removed.
+* The undocumented ``XKILLMY`` command and its associated ``SIGUSR2`` handler
+  have been removed.
+* lipcap support has been removed.
+* The ``tls_server_dhparam`` option has been deprecated.
+* The experimental zeroskip database backend has been removed.
+
+Security fixes
+==============
+
+* None yet!
+
+Significant bugfixes
+====================
+
+* ``lmtpd`` will no longer reset the session id after ``LHLO``.
```

Comment:
I usually style this sort of thing something like:

```
* Fixed :issue:`1234`: :cyrusman:`lmtpd(8)` will no longer [...]
```

These days they're usually PR numbers rather than issue numbers, which is a little weird, but the resultant web link works anyway.